code/_globalvars/soul_glimmer.dm

```diff
@@ -6,12 +6,30 @@
 */
 
 //ALL the psychic soul colours
-GLOBAL_LIST_INIT(SOUL_GLIMMER_COLORS, list("Azure" = "#0091ff", "Vermilion" = "#ff3a1c", "Sage" = "#1fff5e", "Teal" = "#00ffe5", "Terracotta" = "#ffa600", "Champagne" = "#ff009d"))
+GLOBAL_LIST_INIT(soul_glimmer_colors, list(
+	"Azure" = "#0091ff",
+	"Vermilion" = "#ff3a1c",
+	"Sage" = "#1fff5e",
+	"Teal" = "#00ffe5",
+	"Terracotta" = "#ffa600",
+	"Champagne" = "#ff009d",
+	"Sunshine" = "#fff12c",
+	"Melancholia" = "#8100FF",
+	"Submarine" = "#1f1bff",
+	"Humour" = "#e876ff"))
 
-//Low pop colours
-#define SOUL_GLIMMER_LOWER_POP 10
-GLOBAL_LIST_INIT(SOUL_GLIMMER_COLORS_LOW, list("Azure" = "#0091ff", "Vermilion" = "#ff3a1c", "Sage" = "#1fff5e"))
-//mid pop colours
-#define SOUL_GLIMMER_MID_POP 20
-GLOBAL_LIST_INIT(SOUL_GLIMMER_COLORS_MID, list("Teal" = "#00ffe5", "Terracotta" = "#ffa600", "Champagne" = "#ff009d"))
+/// TGUI chat colours
+GLOBAL_LIST_INIT(soul_glimmer_cfc_list, list(
+	"Azure" = "cfc_soul_glimmer_azure",
+	"Vermilion" = "cfc_soul_glimmer_vermilion",
+	"Sage" = "cfc_soul_glimmer_sage",
+	"Teal" = "cfc_soul_glimmer_teal",
+	"Terracotta" = "cfc_soul_glimmer_terracotta",
+	"Champagne" = "cfc_soul_glimmer_champagne",
+	"Sunshine" = "cfc_soul_glimmer_sunshine",
+	"Melancholia" = "cfc_soul_glimmer_melancholia",
+	"Submarine" = "cfc_soul_glimmer_submarine",
+	"Humour" = "cfc_soul_glimmer_humour"))
 
+#define SOUL_GLIMMER_MINIMUM_POP_COLOR 3 // regardless of the pop, 3 colours will always come
+#define SOUL_GLIMMER_POP_COUNT_INTERVAL 4 // Starting from 13, for every INTERVAL value, the round will spawn more soul colour
```

Comment:
This should be higher I think, since this narrows down evidence of interactions by a very significant amount. This should be something like 6 imo

Reply:
actually, I changed my mind. it should be higher.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/87972842/301ba1f0-d44f-4291-bf4c-2d72203ff6d0)